Mathlib/Data/Multiset/Nodup.lean

```diff
@@ -82,13 +82,18 @@ theorem nodup_iff_ne_cons_cons {s : Multiset α} : s.Nodup ↔ ∀ a t, s ≠ a
 theorem nodup_iff_count_le_one [DecidableEq α] {s : Multiset α} : Nodup s ↔ ∀ a, count a s ≤ 1 :=
   Quot.induction_on s fun _l => by
     simp only [quot_mk_to_coe'', coe_nodup, mem_coe, coe_count]
-    apply List.nodup_iff_count_le_one
+    exact List.nodup_iff_count_le_one
 #align multiset.nodup_iff_count_le_one Multiset.nodup_iff_count_le_one
 
+theorem nodup_iff_count_eq_one [DecidableEq α] : Nodup s ↔ ∀ a ∈ s, count a s = 1 :=
+  Quot.induction_on s fun _l => by
+    simp only [quot_mk_to_coe'', coe_nodup, mem_coe, coe_count]
+    exact List.nodup_iff_count_eq_one
```

Comment:
```suggestion
  Quot.induction_on s fun _l => by simpa using List.nodup_iff_count_eq_one
```
if you want, or
```
  by rcases s; simp_rw [quot_mk_to_coe'', coe_count]; exact List.nodup_iff_count_eq_one
```
The actual non-defeq simp lemma used is `coe_count`, because [Multiset.countP](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Data/Multiset/Basic.html#Multiset.countP) uses `Prop` but [List.countP](https://leanprover-community.github.io/mathlib4_docs/Std/Data/List/Basic.html#List.countP) uses `Bool`.